index.html

```diff
@@ -8,6 +8,69 @@
     <link href="./styles.css" rel="stylesheet" >
 </head>
 <body>
-    Bismillah. 
+    <div id="blue-box">
```

Comment:
lines from 11 - 73 should be wrapped in a container right? Since they all form one complete section..